src/server.c

```diff
@@ -97,6 +99,50 @@ static int avoid_dupes(const char *url)
         return retval;
 }
 
+/*
+ * Read URLs from environment variable, space separated
+ */
+int configure_urls(void)
+{
+	const char *env_var = getenv("CLR_DEBUGINFO_URLS");
+	const char *token = env_var;
+	int count = 0;
+	char **urls_new = NULL;
+	if (env_var) while (1) {
+		int token_len = strcspn(token, " \t\n");
+		if (token_len) {
+			count++;
+			urls_new = realloc(urls_new, count * sizeof(char*));
+			if (!urls_new) { perror("realloc()"); exit(EXIT_FAILURE); }
+
+			char *url = calloc(token_len+1, sizeof(char));
+			if (!url) { perror("calloc()"); exit(EXIT_FAILURE); }
+
+			urls_new[count-1] = strncpy(url, token, token_len);
+			token += token_len;
```

Comment:
when you make ```token += token_len;``` the pointer result would point to the space if two or more url are present isn't it. this could represent a problem for the alternative url.

Reply:
The `token++` a few lines below moves the algorithm one more char forward, which should be the start of the next url.

`token` can indeed still land on a space if user enters multiple spaces. In that case `strcspn` returns zero and the extra space is ignored.

Reply:
ok, then this looks good to me